collector/generatorreceiver/internal/topology/latency_percentiles.go

```diff
@@ -75,3 +77,21 @@ func (l *LatencyPercentiles) loadDurations() error {
 	}
 	return nil
 }
+
+type LatencyConfigs []*LatencyPercentiles
+
+func (lcfg *LatencyConfigs) Sample() int64 {
+	var defaultCfg *LatencyPercentiles
+	var enabled []*LatencyPercentiles
+	for _, cfg := range *lcfg {
+		if cfg.IsDefault() {
```

Comment:
I don't really understood this check, shouldn't a config that has `IsDefault() == true` always be `ShouldGenerate() == true` as well? if that is true, I think we can simplify this code and remove the `defaultCfg`?

Reply:
Also, what happens if there are more than one `IsDefault()` config? Don't seem to be blocked during validation.

Reply:
`IsDefault() == true` does imply `ShouldGenerate() == true`, but not the other way around. The idea is that we only want to include a default latency set if none of the ones with flags is enabled.

I probably should add some validation for having exactly one default config, that's a good point.